product_exception/README.rst

```diff
@@ -0,0 +1,114 @@
+=================
+Product Exception
+=================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproduct--attribute-lightgray.png?logo=github
+    :target: https://github.com/OCA/product-attribute/tree/10.0/product_exception
+    :alt: OCA/product-attribute
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/product-attribute-10-0/product-attribute-10-0-product_exception
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/135/10.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to define rules to check products.
+It gives feedback to users about wrongly configured products.
+
+Common use examples:
+- public price is less than its cost (see demo example)
+- manufactured product with no Bill of Materials
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+This module don't provide any menu to edit the rules.
+See base_exception for more information on rules.
+
+Usage
+=====
+
+A button "Check Exception" is present on the header of the product form.
```

Comment:
Just a minor comment, the button now is called "test button"

Reply:
Thanks! :smile: